<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyWindow</class>
 <widget class="QMainWindow" name="MyWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1086</width>
    <height>646</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MyWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QFrame" name="rysujFrame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>800</width>
      <height>600</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QGroupBox" name="exitBox">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>560</y>
      <width>241</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="exitButton">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>241</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Zakończ program</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="layerBox">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>20</y>
      <width>241</width>
      <height>381</height>
     </rect>
    </property>
    <property name="title">
     <string>wybór oraz włączenie warstw</string>
    </property>
    <widget class="QRadioButton" name="layerRadio_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>201</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>(6) Toy-Story</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="layerRadio_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>201</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>(5) Spongebob</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="layerRadio_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>200</y>
       <width>201</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>(3) Young Leosia</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="layerRadio_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>201</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>(4) Spiderman</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="layerRadio_1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>321</y>
       <width>201</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>(1) Zima</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="layerRadio_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>261</y>
       <width>201</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>(2) Książki</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="layerCheckBox_6">
     <property name="geometry">
      <rect>
       <x>215</x>
       <y>42</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QCheckBox" name="layerCheckBox_5">
     <property name="geometry">
      <rect>
       <x>215</x>
       <y>102</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QCheckBox" name="layerCheckBox_4">
     <property name="geometry">
      <rect>
       <x>215</x>
       <y>162</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QCheckBox" name="layerCheckBox_3">
     <property name="geometry">
      <rect>
       <x>215</x>
       <y>222</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QCheckBox" name="layerCheckBox_2">
     <property name="geometry">
      <rect>
       <x>215</x>
       <y>282</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QCheckBox" name="layerCheckBox_1">
     <property name="geometry">
      <rect>
       <x>215</x>
       <y>342</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="Line" name="line_1">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>70</y>
       <width>241</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="Line" name="line_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>130</y>
       <width>241</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="Line" name="line_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>250</y>
       <width>241</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="Line" name="line_4">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>190</y>
       <width>241</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="Line" name="line_5">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>310</y>
       <width>241</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </widget>
   <widget class="QSlider" name="alphaSlider">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>470</y>
      <width>241</width>
      <height>21</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="value">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QComboBox" name="alphaModeBox">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>420</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <item>
     <property name="text">
      <string>Normal Mode</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Darken Mode</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Lighten Mode</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Burn Mode</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Average Mode</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Additive Mode</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="ResetButton">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>520</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Resetuj wartości</string>
    </property>
   </widget>
   <widget class="QLabel" name="krycieLabel">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>485</y>
      <width>241</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Krycie</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
